boot/freeldr/freeldr/options.c

```diff
@@ -277,4 +286,10 @@ VOID AppendBootTimeOptions(PCHAR BootOptions)
 
     if (DebuggingMode)
         strcat(BootOptions, " /DEBUG");
+
+    if (EditCommandLine != NULL)
+    {
+        strcat(BootOptions, " ");
+        strcat(BootOptions, EditCommandLine);
```

Comment:
The other `strcat`s may or may not be safe, but this one is most definitely prone to overflowing. Consider `RtlStringCbCatA` or similar.

Reply:
I don't want to use that way of editing the command-line, but I'll keep that in mind.